src/atomist.config.ts

```diff
@@ -15,8 +15,8 @@
  */
 
 import { Configuration } from "@atomist/automation-client";
-import { ingester } from "@atomist/automation-client/graph/graphQL";
-import * as secured from "@atomist/automation-client/secured";
+import { ingester } from "@atomist/automation-client/lib/graph/graphQL";
+import * as secured from "@atomist/automation-client/lib/secured";
```

Comment:
Can these not be imported from the index?

Reply:
Handled. Thx.